kythe/data/simple_vnames.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "pattern": "bazel-out/[^/]+/bin/external/[^/]+/(.+)",
+    "vname": {
+      "corpus": "CORPUS",
+      "path": "@1@"
+    }
+  },
+  {
+    "pattern": "bazel-out/[^/]+/([^/]+)/external/[^/]+/(.+)",
+    "vname": {
+      "corpus": "CORPUS",
+      "root": "bazel-out/@1@",
```

Comment:
Make it `bazel-out/@1@/external/@repo_name@` so they don't conflict with the repository's generated files and other external repo files.